python/lsst/ts/xml/generate_subsystems_doc.py

```diff
@@ -291,12 +291,19 @@ def generate_subsystems_doc():
                             write_heading(cf, name="Enumerations")
                             enumeration_seen = True
                         states = topic.text.split(",")
+                        print(states)
                         state_name = states[0].split("_")[0].lstrip()
                         cf.write(f":{state_name}:\n")
                         for state in states:
+                            state = state.split("_")[1:]
```

Comment:
Please add a code comment explaining the format of `state`. That should make the code much easier to understand.